server/views/profile.hbs

```diff
@@ -23,6 +23,14 @@
     {{/if}}
   {{/if}}
 
+  <h3> Photos: </h3>
+  
+  <div>
+  {{#each user.photos}}
+    <img src="{{url}}"/>
```

Comment:
Ovo će sada prikazati listu slika, no <img /> element će ih prikazati velicinama onakvih kakve su uploadane, a to ne zelimo jer ne zelimo da nam slike zauzimaju cijeli ekran. Ako stavis width ili height atribut na taj element mozes ogranicniti velicinu slike. Samo pazi da postavis samo height a ne i width jer tada ce se slika spljostiti, tj. izgubit ce aspect ratio